### PR TITLE
Use CSV for boundaries

### PR DESCRIPTION
**Pull request by zerebubuth**
Connects to #646.

@rmarianski could you review, please?

### CODE REVIEW

**Comment by zerebubuth** [comment]
@nvkelso: does this cover all of the NE `featurecla` -> `kind` mapping in the comments of #648?

**Comment by rmarianski** [comment]
:+1:

File: spreadsheets/kind/boundaries.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ne:featurecla,osm:boundary,osm:admin_level,osm:barrier,osm:historic,osm:man_made,min_zoom,kind
+*,administrative,2;4,*,*,*,${8},${NULL}
+*,administrative,6;8,*,*,*,${10},${NULL}
+*,*,*,city_wall,*,*,${12},city_wall
+*,*,*,*,citywalls,*,${12},city_wall
+*,*,*,*,*,snow_fence,${15},snow_fence
+*,*,*,retaining_wall,*,*,${15},retaining_wall
+*,*,*,fence,*,*,${16},fence
+1st Order Admin Lines,*,*,*,*,*,${0},1st_order_admin_lines
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: kind:state

File: spreadsheets/kind/boundaries.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Admin-1 boundary,*,*,*,*,*,${0},admin-1_boundary
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: kind:state

File: spreadsheets/kind/boundaries.csv [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Admin-1 region boundary,*,*,*,*,*,${0},admin-1_region_boundary
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: kind:macroregion

File: spreadsheets/kind/boundaries.csv [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Disputed (please verify),*,*,*,*,*,${0},disputed
+Indefinite (please verify),*,*,*,*,*,${0},indefinite
+Indeterminant frontier,*,*,*,*,*,${0},indeterminant_frontier
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: kind:indeterminate

File: spreadsheets/kind/boundaries.csv [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+International boundary (verify),*,*,*,*,*,${0},international_boundary
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: kind:country

File: spreadsheets/kind/boundaries.csv [hunk 6]
```diff
@@ -0,0 +1,20 @@
+ne:featurecla,osm:boundary,osm:admin_level,osm:barrier,osm:historic,osm:man_made,min_zoom,kind
+*,administrative,2;4,*,*,*,${8},${NULL}
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: is this kind:country?

File: spreadsheets/kind/boundaries.csv [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+*,administrative,6;8,*,*,*,${10},${NULL}
```
Comment (nvkelso) [inline spreadsheets/kind/boundaries.csv]: is this `kind:state`?

File: spreadsheets/kind/boundaries.csv (hunk 6 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: We do a [complex transform](https://github.com/mapzen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/transform.py#L469-L494) in TileStache to get the kind. Created #687 to track.

File: spreadsheets/kind/boundaries.csv (hunk 7 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: Same as above, see #687 for tracking issue.

File: spreadsheets/kind/boundaries.csv (hunk 1 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: Tracked in #687.

File: spreadsheets/kind/boundaries.csv (hunk 2 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: Tracked in #687.

File: spreadsheets/kind/boundaries.csv (hunk 3 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: Tracked in #687.

File: spreadsheets/kind/boundaries.csv (hunk 4 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: Tracked in #687.

File: spreadsheets/kind/boundaries.csv (hunk 5 above)
Comment (zerebubuth) [inline spreadsheets/kind/boundaries.csv]: Tracked in #687.

**Pull request merged**